### ISSUE TITLE
Add createdAt property to NewsletterEmail GQL type

### ISSUE THREAD

**Issue by jacksonh** [omnivore-app/omnivore#1626]
```
  type NewsletterEmail {
    id: ID!
    address: String!
    confirmationCode: String
  }
 ```

looks like we have created_at in the database but just don't expose it in GQL. Adding this will let us show the user on the newsletter page when their email address was added.

**Comment by sywhb** [comment]
Merged #1633

**Comment by jacksonh** [comment]
💥
